data/sql/updates/pending_db_world/rev_1560396171418694700.sql

```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1560396171418694700');
+
+-- Nexus Raid Platform has a lot more hp in order not to break in case any siege damage is taken, initial value was 100
+UPDATE `gameobject_template_addon` SET `Data0`='6000000' WHERE  `entry`=193070;
```

Comment:
Maybe like this?
```suggestion
UPDATE `gameobject_template` SET `Data0`='6000000' WHERE `entry`=193070;
```

Reply:
I'm sorry for that, I'm tired now I've changed and it's okay now